src/core/connection.c

```diff
@@ -3698,6 +3702,10 @@ QuicConnRecvHeader(
 
     CXPLAT_FRE_ASSERT(QuicIsVersionSupported(Connection->Stats.QuicVersion));
 
+    const BOOLEAN IgnoreFixedBit = (Connection->Settings.GreaseQuicBitEnabled &&
+        (QuicConnIsClient(Connection) ||
```

Comment:
Why does client always do it? Put a comment here explaining this?

Reply:
And should we constantly be reevaluating this or should we store it on the connection?

Reply:
I'd prefer to store this state in the connection.

Reply:
If this feature is turned on in the client, it can accept FixedBit as 0 according to RFC since it will send this TP, but the Server cannot ignore this bit without receiving TP.

Reply:
Yeah, we should definitely store it on the connection. In this way, things will get easier to do.